addons/mail/static/src/core/common/out_of_focus_service.js

```diff
@@ -90,7 +90,7 @@ export class OutOfFocusService {
      * @param {string} [param0.icon] The icon to be displayed in the
      * notification.
      */
-    sendNotification({ message, sound = true, title, type, icon }) {
+    sendNotification({ message, sound = false, title, type, icon }) {
```

Comment:
Do we really need to change the default in stable?
It feels like just passing `false` when we don't need it is sufficient.